ibr/png-parser/BUILD

```diff
@@ -0,0 +1,11 @@
+load("@rules_python//python:defs.bzl", "py_binary")
+
+py_binary(
+    name = 'CreateIBR',
```

Comment:
nit: s/CreateIBR.py /create_ibr.py/g, also build target should probably match

pep8 module/file names should be lower/snakecase 

https://www.python.org/dev/peps/pep-0008/#package-and-module-names

ps: we should probably set up a python/bazel linter (separate diff) while we're at it

Reply:
will punt bazel python linter to another PR.

Reply:
sg